persistent-test/src/CompositeTest.hs

```diff
@@ -1,27 +1,15 @@
-{-# OPTIONS_GHC -fno-warn-unused-binds -fno-warn-orphans #-}
 {-# LANGUAGE DeriveGeneric #-}
-{-# LANGUAGE EmptyDataDecls #-}
-{-# LANGUAGE FlexibleContexts #-}
-{-# LANGUAGE FlexibleInstances #-}
-{-# LANGUAGE GADTs #-}
+{-# LANGUAGE ExistentialQuantification #-}
 {-# LANGUAGE GeneralizedNewtypeDeriving #-}
-{-# LANGUAGE MultiParamTypeClasses #-}
-{-# LANGUAGE NoMonomorphismRestriction #-}
-{-# LANGUAGE OverloadedStrings #-}
-{-# LANGUAGE QuasiQuotes #-}
-{-# LANGUAGE ScopedTypeVariables #-}
-{-# LANGUAGE StandaloneDeriving #-}
-{-# LANGUAGE TemplateHaskell #-}
-{-# LANGUAGE TypeFamilies #-}
 {-# LANGUAGE UndecidableInstances #-} -- FIXME
+{-# OPTIONS_GHC -Wno-incomplete-uni-patterns #-}
```

Comment:
this is a great warning though!

Reply:
I added it, because it's used a ton with the idea that, if it doesn't match, the test would fail (because of the partial match), so no reason to have all those warnings flood the screen, right?
I can remove it if you prefer.

Reply:
It's fine to have this warning flag still in. A later refactoring could remove the need for it.